Zend/zend_language_parser.y

```diff
@@ -796,8 +796,8 @@ attributed_class_statement:
 		variable_modifiers optional_type_without_static property_list ';'
 			{ $$ = zend_ast_create(ZEND_AST_PROP_GROUP, $2, $3, NULL);
 			  $$->attr = $1; }
-	|	method_modifiers T_CONST class_const_list ';'
-			{ $$ = zend_ast_create(ZEND_AST_CLASS_CONST_GROUP, $3, NULL);
+	|	method_modifiers T_CONST type_expr_without_static class_const_list ';'
```

Comment:
TODO: This needs to be `optional_type_without_static` instead of `type_expr_without_static` but since there are 4 shift/reduce conflicts using the former one, I am temporarily using `type_expr_without_static`.